torchvision/transforms/functional_tensor.py

```diff
@@ -1,38 +1,45 @@
+from __future__ import division
 import torch
 import torchvision.transforms.functional as F
+from torch import Tensor
+from torch.jit.annotations import Optional, List, BroadcastingList2
+from typing import Tuple
 
 
-def vflip(img_tensor):
+def vflip(img):
+    # type: (Tensor) -> Tensor
     """Vertically flip the given the Image Tensor.
 
     Args:
-        img_tensor (Tensor): Image Tensor to be flipped in the form [C, H, W].
+        img (Tensor): Image Tensor to be flipped in the form [C, H, W].
 
     Returns:
         Tensor:  Vertically flipped image Tensor.
     """
-    if not F._is_tensor_image(img_tensor):
+    if not len(img.shape) == 3:
```

Comment:
Can we instead define `_is_image_tensor` in `functional_tensor`? This way we can use it and have richer checks

Reply:
Sure, that would be good.